### ISSUE TITLE
Port to Godot Engine 4 with its autolod

### ISSUE THREAD

**Issue by fire** [stuarta0/launceston-3d-godot#1]
This project is cool.

I'm looking for test projects to trial Godot Engine 4's auto lod system.

**Comment by stuarta0** [comment]
Yes this is my primary goal - to have a test case for Godot 4's new features like SDFGI, LOD and (shortly) occlusion culling
